README.md

```diff
@@ -28,9 +25,9 @@ To do this follow the instructions in the
 
 ### Optional Installs
 
-* **PyTorch**, may be installed either using command `pip install 'qiskit-machine-learning[torch]'` to install the
-  package or refer to PyTorch [getting started](https://pytorch.org/get-started/locally/). When PyTorch
-  is installed, the `TorchConnector` facilitates its use of quantum computed networks.
+* **PyTorch**, may be installed either using the command `pip install 'qiskit-machine-learning[torch]'` to install the
```

Comment:
The addition of 'the' is ok, seems like it was missed out. Thanks for spotting that. Lets not add text referring to someone as 'one'. Also I think 'its' in the last part is more correct the TorchConnector facilitates the use of quantum networks within/by Torch, the subject of this paragraph, and to which 'it' is referencing